modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -178,9 +184,38 @@ class ONNXImporter
     const std::string str_domain_ai_onnx = "ai.onnx";
 };
 
+class ONNXLayerHandler : public detail::LayerHandler
+{
+public:
+    explicit ONNXLayerHandler(ONNXImporter* importer_);
+
+    void fillRegistry(const opencv_onnx::GraphProto& net);
+
+protected:
+    ONNXImporter* importer;
+};
+
+ONNXLayerHandler::ONNXLayerHandler(ONNXImporter* importer_) : importer(importer_){}
+
+void ONNXLayerHandler::fillRegistry(const opencv_onnx::GraphProto &net)
+{
+    int layersSize = net.node_size();
+    for (int li = 0; li < layersSize; li++) {
+        const opencv_onnx::NodeProto &node_proto = net.node(li);
+        const std::string& name = node_proto.output(0);
+        const std::string& type = node_proto.op_type();
+        const std::string& layer_type_domain = importer->getLayerTypeDomain(node_proto);
+        const auto& dispatch = importer->getDispatchMap(node_proto);
+        if (dispatch.find(type) == dispatch.end())
+        {
+            addMissing(name, cv::format("%s::%s", layer_type_domain.c_str(), type.c_str()));
```

Comment:
Need to align format of used "type"s:

With disabled quantized labels and resnet50_int8.onnx opencv_model_diagnostics tool emits this
```
...
Type='com.microsoft::QLinearGlobalAveragePool', affected nodes:
['resnetv17_pool1_fwd_quantized']
...
OpenCV(4.5.5-dev) modules/dnn/src/dnn.cpp:621: error: (-2:Unspecified error) Can't create layer "resnetv17_pool1_fwd_quantized" of type "com.microsoft.QLinearGlobalAveragePool" in function 'getLayerInstance'
```

- `com.microsoft::QLinearGlobalAveragePool`
- `com.microsoft.QLinearGlobalAveragePool`

Such misalignment would confuse users if they want to implement own custom layer.